Sources/_StringProcessing/Engine/MEBuiltins.swift

```diff
@@ -1,13 +1,221 @@
-
+@_implementationOnly import _RegexParser // For AssertionKind
 
 extension Processor {
+  mutating func matchBuiltin(
+    _ cc: _CharacterClassModel.Representation,
+    _ isInverted: Bool,
+    _ isStrictAscii: Bool
+  ) -> Bool {
+    guard let c = load() else {
+      signalFailure()
+      return false
+    }
+
+    var matched: Bool
+    var next = input.index(after: currentPosition)
+    switch cc {
+    case .any, .anyGrapheme: matched = true
+    case .anyScalar:
+      matched = true
+      next = input.unicodeScalars.index(after: currentPosition)
+    case .digit:
+      matched = c.isNumber && (c.isASCII || !isStrictAscii)
+    case .horizontalWhitespace:
+      matched = c.unicodeScalars.first?.isHorizontalWhitespace == true
```

Comment:
Could you write this as an underscored property on `Character`, so that it follows the character properties model (even if it's not public)?